blaze/compute/tests/test_sql_compute.py

```diff
@@ -1508,3 +1538,29 @@ def test_not():
         accounts.name not in (:name_1, :name_2)
     """
     assert normalize(result) == normalize(expected)
+
+
+@pytest.yield_fixture
+def sql_with_dts(url):
+    try:
+        t = resource(url, dshape='var * {A: datetime}')
+    except sa.exc.OperationalError as e:
+        pytest.skip(str(e))
+    else:
+        t = odo([(d,) for d in pd.date_range('2014-01-01', '2014-02-01')], t)
+        try:
+            yield t
+        finally:
+            drop(t)
+
+
+def test_timedelta_arith(sql_with_dts):
+    delta = timedelta(days=1)
+    dates = pd.Series(pd.date_range('2014-01-01', '2014-02-01'))
+    sym = symbol('s', '32 * datetime')
```

Comment:
you should be able to do `symbol('s', discover(dates))` instead of explicitly writing down the dshape